public/components/query_compare/search_result/result_components/result_panel.tsx

```diff
@@ -47,7 +56,9 @@ export const ResultPanel = ({ resultNumber, queryResult }: ResultPanelProps) =>
           </EuiTitle>
         </EuiFlexItem>
       </EuiFlexGroup>
-      {queryResult?.hits?.hits?.length ? (
+      {queryError.errorResponse.statusCode !== 200 || queryError.queryString.length ? (
+        ErrorMessage
+      ) : queryResult?.hits?.hits?.length ? (
```

Comment:
is it possible that there's non-empty query error together with query result?

Reply:
I believe the way `/api/relevancy/search` is set up, the [request](https://github.com/opensearch-project/dashboards-search-relevance/blob/26f1ea0651fe6422544fb0f501a0c3dc9fcae4da/server/routes/dsl_route.ts#L99-L115) doesn't set search results to return if an error is caught.